test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -93,7 +93,7 @@ UCS_TEST_P(test_ib_md, ib_md_umr_rcache_on, "RCACHE=yes") {
     free(rkey_buffer);
 }
 
-UCS_TEST_P(test_ib_md, ib_md_umr_rcache_off, "RCACHE=no") {
+UCS_TEST_P(test_ib_md, ib_md_umr_rcache_off, "REG_METHODS=direct") {
 
```

Comment:
ODP case ?

Reply:
Still, do we have to test ODP explicitly ?

Reply:
currently UMR is disabled for global ODP key, and this test tests UMR